### PR TITLE
MOO-306: Baseline NCD Forms 

### PR DESCRIPTION
**Pull request by ssmusoke**

### CODE REVIEW

File: omod/src/main/webapp/resources/htmlforms/clinicalNote.xml [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+		</div>
+	</div>
+
+	<!-- Ensure that only one version of this form can exist at any one time -->
+	<ifMode mode="EDIT" include="false">
+		<lookup complexExpression="
+            #set( $encounter = 0 )
+            #set( $encounter = $fn.allEncounters('fa6f3ff5-b784-43fb-ab35-a08ab7dbf074'))
+            #foreach($encounter in $fn.allEncounters(null)) $encounter.form.uuid
+                #if( $encounter.form.uuid == '30d1fda4-4161-4666-ad0c-e2ba20eb73a6' )
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/clinicalNote.xml]: @tmvumbi2 Change this uuid to the uuid of this form

File: omod/src/main/webapp/resources/htmlforms/clinicalNote.xml [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+
+				<section>
+
+					<h3>Clinical note</h3>
+
+					<table class="baseline-aligned">
+						<tr>
+							<td>
+								<!-- TODO update the concept -->
+								<obs conceptId="463346" style="textarea" rows="3"/>
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/clinicalNote.xml]: @tmvumbi2 Needs a label

File: omod/src/main/webapp/resources/htmlforms/clinicalNote.xml [hunk 3]
```diff
@@ -0,0 +1,148 @@
...
+						<tr>
+							<td><lookup expression="fn.getConcept('463571').name"/>:
+							<obs conceptId="463571" id="patient-education"/></td>
+						</tr>
+
+						<tr>
+							<td>
+								<!-- TODO update the concept -->
+								<obsgroup groupingConceptId="463603">
+									<obs conceptId="463603" answerConceptId="463605" style="checkbox" />
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/clinicalNote.xml]: @tmvumbi2 Needs a label

File: omod/src/main/webapp/resources/htmlforms/clinicalNote.xml [hunk 4]
```diff
@@ -0,0 +1,148 @@
...
+									<obs conceptId="463603" answerConceptId="463607" style="checkbox" />
+									<obs conceptId="463603" answerConceptId="463608" style="checkbox" />
+								</obsgroup>
+							</td>
+						</tr>
+
+						<tr>
+							<td>
+								<!-- TODO update the concept -->
+								<obs conceptId="463347" style="textarea" rows="3"/>
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/clinicalNote.xml]: @tmvumbi2 Needs a label

File: omod/src/main/webapp/resources/htmlforms/clinicalNote.xml [hunk 5]
```diff
@@ -0,0 +1,148 @@
+<htmlform
+	formUuid="f09a3a3a-810e-4cf6-b432-3d43da303948"
+	formName="Clinical Note"
```
Comment (ssmusoke) [inline omod/src/main/webapp/resources/htmlforms/clinicalNote.xml]: @tmvumbi2 to be consistent can we use NCD Baseline Consultation - Clinical Note

File: omod/src/main/webapp/fragments/leftMenu.gsp [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+        <ol>
+            <li>
+                <a id="medicalhistory" href="#medicalhistory">Medical history</a>
+                <span class="hidden" id="medical-history-form-url">enterHtmlFormWithStandardUi.page?formUuid=06807e2b-ce97-4d65-8796-e955fcbe057d</span>
+            </li>
+            <li>
+                <a id="lifestyle" href="#lifestyle">Lifestyle</a>
+                <span class="hidden" id="lifestyle-form-url">enterHtmlFormWithStandardUi.page?formUuid=3209cd5f-656e-42f4-984e-ab466a5b77ef</span>
+            </li>
+            <li class="active">
```
Comment (ssmusoke) [inline omod/src/main/webapp/fragments/leftMenu.gsp]: @tmvumbi2 Remove this class active so that it can be set on a page by page basis

Comment (tmvumbi2) [reply omod/src/main/webapp/fragments/leftMenu.gsp]: Thx @ssmusoke, see fix here: https://github.com/jembi/openmrs-module-msfcore/pull/22/commits/0b8df3f3e984f1072901d804f7a82cedf2a8e8c4

**Pull request closed without merging**